tests/plugins/emoji/manual/editablenullerror.md

```diff
@@ -0,0 +1,9 @@
+@bender-tags: 4.16.2, bug, emoji, 4782
+@bender-ckeditor-plugins: wysiwygarea, sourcearea, toolbar, emoji 
+@bender-ui: collapsed
+
+1. Open browser's dev console.
+1. Switch to `source` mode.
+1. Switch to `wysiwyg` mode.
+
+**Expected** No errors in the console.
```

Comment:
I somehow missed that in the previous review... and it's good that this test covers the initial issue (error) but maybe it could have one more step to open emoji panel so we can see if nothing breaks? ~Unless we are 100% sure that...~ Well, because this change affects how autocomplete panel (and so emoji) works so maybe let's have step covering regressions too?

Reply:
After F2F talks we decided to leave it as is :+1: